editor/plugins/org.fusesource.ide.wsdl2rest.ui/.classpath

```diff
@@ -64,7 +19,6 @@
 	<classpathentry exported="true" kind="lib" path="libs/commons-beanutils.jar"/>
 	<classpathentry exported="true" kind="lib" path="libs/commons-collections.jar"/>
 	<classpathentry exported="true" kind="lib" path="libs/commons-digester.jar"/>
-	<classpathentry exported="true" kind="lib" path="libs/commons-lang.jar"/>
```

Comment:
I think that commons-lang needs to be kept.
Both commons-lang and commons-lang3 are required by 2 different dependencies

![image](https://user-images.githubusercontent.com/1105127/57068096-089d4500-6cd1-11e9-83bf-71e5c7485a2e.png)